bravado/http_client.py

```diff
@@ -197,6 +197,9 @@ def __init__(self, session, request):
         self.session = session
         self.request = request
 
+    def result(self, *args, **kwargs):
```

Comment:
This was added to unify interface of sync and async clients (as `fido` doesn't expose `wait` method).